pkg/virt-handler/vm_test.go

```diff
@@ -1632,7 +1632,8 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Iface: &v1.Interface{
 					Name: interfaceName,
 				},
-				PodIP: "1.1.1.1",
+				PodIP:  "1.1.1.1",
+				PodIPs: [2]string{"1.1.1.1", "fd10:244::8c4c"},
```

Comment:
That's a wrong type. You created an array while it should be a slice.

Reply:
https://github.com/kubevirt/kubevirt/pull/3958/files/5d1e3b364c04018d3c14acb080f5ed3846fe002b#diff-5e37ab6e114c3110d75e482145f60f8fR49

its in the cache, a new field i added, not the slice that was exist already in the DST struct
https://github.com/kubevirt/kubevirt/pull/3958/files/5d1e3b364c04018d3c14acb080f5ed3846fe002b#diff-27207346391eaa19190c76760eb8851fR529

Reply:
I you choose the approach I suggested, the 2 here has to be removed.

Reply:
known thx